src/utils/common/common.ts

```diff
@@ -1,4 +1,5 @@
-import React from 'react';
+import React, { MutableRefObject, Ref } from 'react';
+import { getLogger } from '../..';
```

Comment:
I suggest that we should use explicit path for the logger.

```tsx
import { getLogger } from '../log/logger';
```

Reply:
Hmm.. interesting. When changing this to suggested `import { getLogger } from '../log/logger';` following snapshots will have changes:

```
src/core/Expander/Expander/__snapshots__/Expander.test.tsx.snap
src/core/Expander/ExpanderGroup/__snapshots__/ExpanderGroup.test.tsx.snap
src/core/Expander/ExpanderTitle/__snapshots__/ExpanderTitle.test.tsx.snap
src/core/Form/Checkbox/__snapshots__/CheckboxGroup.test.tsx.snap
src/core/Form/RadioButton/__snapshots__/RadioButtonGroup.test.tsx.snap
src/core/Form/SearchInput/__snapshots__/SearchInput.test.tsx.snap
src/core/Form/Select/MultiSelect/MultiSelect/__snapshots__/MultiSelect.test.tsx.snap
src/core/Form/Select/SingleSelect/__snapshots__/SingleSelect.test.tsx.snap
```